pytest/tests/sanity/upgradable.py

```diff
@@ -90,22 +90,6 @@ def main():
     assert 'error' not in res, res
     assert 'Failure' not in res['result']['status'], res
 
-    # hex_account_id = (b"I'm hex!" * 4).hex()
-    hex_account_id = '49276d206865782149276d206865782149276d206865782149276d2068657821'
```

Comment:
Keep this line. You'll need it to check this based on the latest.

Might need to update `nonce`